src/libraries/Microsoft.Extensions.Logging.Console/src/AnsiParser.cs

```diff
@@ -128,6 +125,8 @@ public void Parse(string message)
             }
         }
 
+        private static bool IsDigit(char c) => (uint)((int)c - (int)'0') <= (uint)('9' - '0');
```

Comment:
```suggestion
        private static bool IsDigit(char c) => (uint)(c - '0') <= ('9' - '0');
```
Is enough here.

Reply:
yes it is. why not?

---
In reply to: [458355352](https://github.com/dotnet/runtime/pull/39729#discussion_r458355352) [](ancestors = 458355352)